src/interactors/interactors.test.js

```diff
@@ -0,0 +1,32 @@
+import Cloud, * as cloudMocks from '@cesarbr/knot-fog-connector-knot-cloud';
+import RegisterDevice from './RegisterDevice';
+
+jest.mock('@cesarbr/knot-fog-connector-knot-cloud');
+
+const mockThing = {
+  id: 'abcdef1234568790',
+  name: 'my-device',
+};
+
+describe('Interactors', () => {
+  beforeEach(() => {
+    cloudMocks.mockAddDevice.mockClear();
+    cloudMocks.mockRemoveDevice.mockClear();
+    cloudMocks.mockUpdateSchema.mockClear();
+    cloudMocks.mockPublishData.mockClear();
+  });
+
+  test('registerDevice: should execute interactor correctly when register a new thing on cloud', async () => {
+    const cloud = new Cloud();
+    const registerDevice = new RegisterDevice(cloud);
+    await registerDevice.execute(mockThing);
+    expect(cloudMocks.mockAddDevice).toHaveBeenCalled();
+  });
+
+  test('registerDevice: should execute interactor without errors when fail to register a new thing', async () => {
```

Comment:
Suggestion :point_down: 

```suggestion
  test('registerDevice: should execute interactor without errors when the registration of a new thing fails', async () => {
```

Reply:
Done!